LibM.Services/Repositories/StudentRepository.cs

```diff
@@ -0,0 +1,32 @@
+﻿using LibM.Contracts.Student;
+using LibM.Data.Access;
+using LibM.Data.Access.Managers;
+using Microsoft.EntityFrameworkCore;
+
+namespace LibM.Services.Repositories
+{
+    public class StudentRepository
+    {
+        private readonly LibMDbContext context;
+
+        public StudentRepository()
+        {
+            context = ContextManager.GetDbContext();
+        }
+
+        public List<StudentViewDto> GetAllStudent()
+        {
+            var students = context.Students.Select(x => new StudentViewDto { 
+                Id = x.Id,
+                FirstName = x.FirstName,
+                LastName = x.LastName, 
+                BirthDate = x.BirthDate, 
+                Gender = x.Gender, 
+                Grade = x.Grade,
+                TransCount = context.Transactions.Count(y => y.StudentId == x.Id)
```

Comment:
Her bir transaction kayıdında ki her bir TransCount için bir db isteği yolluyor. Tabi algoritmik olarak bir sorun yok. eline sağlık daha performanlı hallerini öğreneceğiz.Konuşacağız